code/modules/mob/living/simple_animal/friendly/farm_animals.dm

```diff
@@ -145,18 +145,20 @@
 	var/obj/item/udder/udder = null
 	gold_core_spawnable = FRIENDLY_SPAWN
 	blood_volume = BLOOD_VOLUME_NORMAL
-	food_type = list(/obj/item/food/grown/wheat)
-	tame_chance = 25
-	bonus_tame_chance = 15
 	footstep_type = FOOTSTEP_MOB_SHOE
 	pet_bonus = TRUE
 	pet_bonus_emote = "moos happily!"
 
 /mob/living/simple_animal/cow/Initialize()
 	udder = new()
 	add_cell_sample()
+	TameableCowComponent()
 	. = ..()
 
+///wrapper for the tameable component addition so you can have non tamable cow subtypes
+/mob/living/simple_animal/cow/proc/TameableCowComponent()
```

Comment:
Rename this to `make_tameable` so it's 1. snake_case and 2. an actual name of action.